src/Initializer/tree/LTSTree.hpp

```diff
@@ -195,7 +195,8 @@ class seissol::initializers::LTSTree : public seissol::initializers::LTSInternal
     }
 
     for (size_t id = 0; id < scratchpadMemSizes.size(); ++id) {
-      assert((scratchpadMemSizes[id] > 0) && "ERROR: scratchpad mem. size is equal to zero");
+      // TODO {ravil}: check whether the assert makes sense
+      //assert((scratchpadMemSizes[id] > 0) && "ERROR: scratchpad mem. size is equal to zero");
```

Comment:
How much work is it to check this?
Should we keep it commented in, until you have checked that?

Reply:
I guess it will take a while. I would leave the comment. The assert seems to have no sense. After this PR, @VMPW will be able to add elastic-convergence to gitlab CI pipeline.